plugins/nexus_trinity_plugin.py

```
@@ -0,0 +1,396 @@
+        code       = payload.get("code", "")
+        print(f"\n[nexus] Gate 1 — Hypothesis: {hypothesis[:80]}")
+        result = self._run_gate(1, hypothesis=hypothesis, contract=contract, code=code)
+        self._bus.dispatch("nexus.gate_result", result)
+
+    def _run_pipeline(self, payload):
+        hypothesis = payload.get("hypothesis", "")
+        contract   = payload.get("contract_path", "")
+        code       = payload.get("code", "")
+        gates      = payload.get("gates", list(range(1, 8)))
+
+        print(f"\n[nexus] ══ 7-GATE PIPELINE ══")
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)